lib/OrdinaryDiffEqPDIRK/src/algorithms.jl

```diff
@@ -1,10 +1,17 @@
-################################################################################
-
-#################################################
-"""
-PDIRK44: Parallel Diagonally Implicit Runge-Kutta Method
-A 2 processor 4th order diagonally non-adaptive implicit method.
-"""
+@doc differentiation_rk_docstring("A 2 processor 4th order diagonally non-adaptive implicit method.",
+    "PDIRK44",
+    "Parallel Diagonally Implicit Runge-Kutta Method.";
+    references = "TBD",
```

Comment:
Might have looked over it in your general explanation of PDIRK, but the method still needs a reference, @ChrisRackauckas.

Reply:
https://github.com/SciML/OrdinaryDiffEq.jl/pull/815 https://academic.oup.com/imajna/article-abstract/10/4/463/697696?redirectedFrom=fulltext&login=false